src/Core/Orm/DB.php

```diff
@@ -4,158 +4,180 @@
 use PDOException;
 use PDO;
 use FFI\Exception;
-use Wepesi\Core\Config;
-
-class DB
+    class DB
```

Comment:
This class has been review for better perfomances ans structure, for better maintenance